src/ER_V/acl/acl.h

```diff
@@ -14,25 +14,28 @@
 
 #define ACL_CMD_SIZE 32
 
-#define ACL_DEFAULT_COMMAND_DELAY_USEC 200000
+#define ACL_DEFAULT_COMMAND_DELAY_MS 200
+#define ACL_DEFAULT_HOME_DELAY_MS 30000
```

Comment:
I changed a couple things about how ACL commands are sent to the arm.
- WriteCommandQueue now enqueues commands into an async buffer, allowing Scorbot to do other things while commands are waiting
- ACL Commands now have a delay_ms field, which defaults to 200ms, but can be overridden for commands than we know take more or less time (abort is processed much faster than shift or move, and home takes much longer than anything else)
- The home delay ended up not being necessary, because if you issue a 'here' command during the homing process, it gets set to the home position (regardless of where the arm is at that particular time)